lib/constants/index.ts

```diff
@@ -135,7 +135,7 @@ export const urls = {
   loveletter: "https://loveletter.klimadao.finance",
   mediaImage: "https://www.klimadao.finance/og-media.png",
   lifiStake:
-    "https://transferto.xyz/embed/stake-klima?fromChain=eth&fromToken=0x0000000000000000000000000000000000000000&toChain=pol&toToken=0x2791bca1f2de4661ed88a30c99a7a9449aa84174",
```

Comment:
nope there should be two,
`lifiStake` for /#/stake
and `lifiOffset` for /#/offset

Reply:
added lifiOffset link (https://develop.transferto.xyz/embed/carbon-offset?fromChain=eth&fromToken=0x0000000000000000000000000000000000000000&toToken=0x2791bca1f2de4661ed88a30c99a7a9449aa84174) and added lifi stake back